proto/pairing/epoch_payments.proto

```diff
@@ -6,6 +6,6 @@ import "pairing/provider_payment_storage.proto";
 
 message EpochPayments {
   string index = 1; 
-  repeated ProviderPaymentStorage clientsPayments = 2; 
+  repeated string providerPaymentStorageKeys = 2; 
```

Comment:
dont use the same protobuf number, reserve 2 and make it the next available

Reply:
done